plugins/cc/cctac.ml

```diff
@@ -43,6 +43,15 @@ let whd env sigma t =
 let whd_delta env sigma t =
   Reductionops.clos_whd_flags CClosure.all env sigma t
 
+let whd_in_concl =
```

Comment:
Use `Tactics.reduct_in_concl`.

Reply:
Ok (I presume `reduct_in_concl ~check:false (Reductionops.whd_all, DEFAULTcast)`).
Since I'm new to Coq's ltac internals, it is not obvious for me which functions are discouraged.

Reply:
I'm not sure why `reduct_in_concl` broke `test-suite:base+async` whereas the previous manual `Reductionops.clos_whd_flags CClosure.all` worked fine. I cannot reproduce the error.

Reply:
The async tests (iirc) outputs different things depending on some internal parallelism. If one of `Reductionops.clos_whd_flags CClosure.all` or `reduct_in_concl` is faster than the other, this may change the probabilty that one output comes after the other.